app/index.js

```diff
@@ -103,8 +103,8 @@ module.exports = class extends Generator {
         const folders = [
             '.Website',
             '.Website/Assets',
+            '.Website/Assets/Fonts',
```

Comment:
What else is supposed to be in the `Assets` folder? Maybe we should move `img` (`Images`) down there too?

Reply:
Good idea, I missed that.. but I agree images should also be moved there.

Reply:
In this case should `images` gulp task point to `Assets/Images` too? Currently it's set to `${config.css.src}/**/img`.

Reply:
hehe.. nice catch I remembered it this morning and think we should probably just remove it. It doesn't do much these days anyway. Before it gave meaning as it was also responsible for compressing and optimizing images now we don't do that as in most cases that's done by zeplin or other tools as such. 🙆‍♂️

Reply:
PS: Committed code removing the image task from gulp because of the reason described above.